components/SVGMapUL.tsx

```diff
@@ -1,8 +1,15 @@
 import React from "react";
-
 import Svg, { G, Path, Text, TSpan } from "react-native-svg";
+import { getHightLightColorForShop } from "../helpers/hightlighting";
+import { SHOP_LIST } from "../constants/constants";
+import { SvgComponentProps as Props } from '../types/types';
+
+// 2do: Fix issue with TS complaining on typing of Text component from
```

Comment:
an option here is to `import Svg, { G, Path, Text as SVGText, TSpan } from "react-native-svg";`

Reply:
I already tried that, but it did not work.

I'm starting to think that it is something with my VSCode, the `react-native-paper`'s `Button` stopped complaining for some reason.

I will review the status again when I perform the fixes.

Reply:
Yeah, VScode can get hung up on types. I often need to restart it to clear errors.